e2e/test-resources/sample-repo/environement_variables/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@rules_java//java:defs.bzl", "java_library", "java_test")
+
+java_library(
+    name = "java_library",
+    srcs = ["JavaEnv.java"],
+    env = {"foo1": "foo2"},
```

Comment:
it's available in binary / test not in libabry


plus u can set these variables in tests (for env_iherit as well, otherwise inherited envs would be empty) in e2e bash scripts https://github.com/JetBrains/bazel-bsp/blob/master/e2e/runTest.sh <- u can add an optional field with envs set for the execution